src/python_testing/basic_composition_support.py

```diff
@@ -167,6 +185,12 @@ async def setup_class_helper(self, allow_pase: bool = True):
         logging.info("Start of actual tests")
         logging.info("###########################################################")
 
+        have_arl, have_carl = arls_populated(self.endpoints_tlv)
```

Comment:
how many tests use this method? Wondering if this will cause test cases to fail when the ARL attribute is cleared.

Reply:
We need to make sure that every test script can pass when the ARL attributes are present but empty.

Reply:
This is used by TC_DeviceConformance, TC_DeviceBasicComposition and TC_AccessControl, so a lot. The function returns false when the attributes are missing or empty. See TestConformanceTest.py for a unit test showing the MACL feature on, but the lists unpopulated.